dev/release/run-rat.sh

```diff
@@ -18,10 +18,14 @@
 # under the License.
 #
 
+RAT_VERSION=0.12
```

Comment:
I always recommend using the form `: ${RAT_VERSION:=0.12}` as it allow you to test the script without changes, e.g. `$ RAT_VERSION=0.13 run-rat.sh` will test the script with `0.13 instead.